tests/test_nlx_directory.py

```diff
@@ -0,0 +1,55 @@
+import os
+from importlib import reload
+from unittest import mock
+
+from django.test import TestCase
+
+from testapp import settings
+from zgw_consumers.constants import NLXDirectories
+from zgw_consumers.models import NLXConfig
+
+
+class NLXConfigEnvvarTestCase(TestCase):
+    custom_demo = "https://demo.custom.nlx.example/"
+    custom_prod = "https://prod.custom.nlx.example/"
```

Comment:
Let's use the function based tests for this (since we're using pytest for this), you can check the other test files to see how this is done